experimental/delegate/mlp_aie_bf16_plugin.cpp

```diff
@@ -47,7 +49,8 @@
 // Uncomment the kernel to use
 // #define DELEGATE_KERNEL_TO_USE REF_MATMUL_DELEGATE_KERNEL
 // #define DELEGATE_KERNEL_TO_USE OPT_DELEGATE_KERNEL
-#define DELEGATE_KERNEL_TO_USE LARGE_MATMUL_DELEGATE_KERNEL
+// #define DELEGATE_KERNEL_TO_USE LARGE_MATMUL_DELEGATE_KERNEL
```

Comment:
same as line above, probably forgot to remove?

Reply:
I know they all look the same, but one is OPT and the other is LARGE
:-D

Reply:
I mean...

#define DELEGATE_KERNEL_TO_USE LARGE_MATMUL_DELEGATE_KERNEL
// #define DELEGATE_KERNEL_TO_USE LARGE_MATMUL_DELEGATE_KERNEL

Reply:
Actually, maybe I misinterpreted what you were saying.  I commented out the LARGE_MATMUL in favor of the next line for the 16k matmul.  The diff view is confusing, so I had to look at the original source to make sure I did it right!

Reply:
Ah, sorry, our comments crossed paths (didn't see your latest).  So this is what is actually says, as seen from a freshly updated workspace:

```
// Uncomment the kernel to use
// #define DELEGATE_KERNEL_TO_USE REF_MATMUL_DELEGATE_KERNEL
// #define DELEGATE_KERNEL_TO_USE OPT_DELEGATE_KERNEL
// #define DELEGATE_KERNEL_TO_USE LARGE_MATMUL_DELEGATE_KERNEL
#define DELEGATE_KERNEL_TO_USE MATMUL_16K_DELEGATE_KERNEL
```

So it's just an GH diff optical illusion :-D